File: app/src/main/java/com/duongtung/cookingman/callback/RecipeAPIs.java
package com.duongtung.cookingman.callback;

import com.duongtung.cookingman.model.Recipe;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface RecipeAPIs {
    @GET("getRecipe.php")
    Call<List<Recipe>> getdataRecipe();
}

File: app/src/main/java/com/duongtung/cookingman/service/RecipeAPIs.kt
package com.duongtung.cookingman.service

import com.duongtung.cookingman.model.Recipe
import io.reactivex.Observable
import retrofit2.http.GET

interface RecipeAPIs {
    @GET("getRecipe.php")
    fun getDataRecipe() : Observable<MutableList<Recipe>>
}
